Formula/inetutils.rb

```diff
@@ -49,6 +49,29 @@ def install
     end
   end
 
+  def caveats
+    if build.without? "default-names"
```

Comment:
IMO, just use a `return if` statement rather than wrapping everything inside the `if`. Looks cleaner.